ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -125,6 +125,7 @@ public bool IsReloadOnJavaScriptChangeEnabled
             }
         }
 
+        //TODO: Implement an abstraction or static helper to save settings based on the platform... The GetSetting and SetSetting methods or their contents are replicated elsewhere...
         private T GetSetting<T>(string key, T defaultValue)
```

Comment:
Can we add a GitHub issue instead of a TODO, can leave a shorter TODO with reference to GitHub issue number